inventory-tracker-gradle/src/main/java/org/launchcode/inventorytrackergradle/models/data/Item.java

```diff
@@ -0,0 +1,60 @@
+package org.launchcode.inventorytrackergradle.models.data;
+
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class Item extends AbstractEntity{
+
+        @ManyToOne
+        private Manufacturer manufacturer;
+
+        private String description;
+
+        private double numberInInventory;
```

Comment:
Are we going to allow non-integer (decimal) numbers for items in inventory?

Reply:
Again, happy to change it, but I decided to set it up this way thinking our fictional client may want to use decimals in their inventory tracker.

Reply:
It's just some input but I think for now dealing with whole numbers may make things easier. You can add decimals to scale once things are up and running. Just a thought.

Reply:
Will do int!